src/main/java/mate/academy/hw3/Sortirivka.java

```diff
@@ -0,0 +1,45 @@
+package mate.academy.hw3;
+
+public class Sortirivka {
```

Comment:
Sortirivka  is not valid name

Reply:
https://github.com/MateAcademy/lessons/pull/16/commits/abb13fd80c57622a18c379c1558c5e8df71fd998